tests/spread/integration/util-linux/task.yaml

```
@@ -130,7 +130,7 @@ execute: |
     ;;
     namespace)
       chroot "${rootfs}" lsns | grep "init"
-      chroot "${rootfs}" nsenter bash -c "exit 0"
+      chroot "${rootfs}" nsenter -t $$ --mount --uts --ipc --net --pid bash -c 'exit 0'
```

```suggestion
      chroot "${rootfs}" nsenter -t $$ --mount --uts --ipc --net --pid bash -c "exit 0"
```

I'd keep these double quotations to make it consistent with the other tests.

---

Good catch, thanks!